### PR TITLE
master-qa-112311

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LPS-112311

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase [hunk 1]
```diff
@@ -20,6 +20,43 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test CheckAUIPreloadConfiguration {
+		property test.name.skip.portal.instance = "UIInfrastructureUsecase#CheckAUIPreloadConfiguration";
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase:25]: Let's not skip portal instance as much as possible. We'll need to include a proper teardown for this testcase. I realize the other testcases don't have a proper teardown as well, will need to address in a separate ticket.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase [hunk 2]
```diff
@@ -20,6 +20,43 @@ definition {
...
+			configurationCategory = "Third Party",
+			configurationName = "AUI",
+			configurationScope = "System Scope");
+
+		FormFields.viewCheckboxChecked(fieldName = "enableAUIPreload");
+
+		SystemSettings.configureSystemSetting(
+			enableSetting = "false",
+			settingFieldName = "Preload Widely Used AUI Modules");
+
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase:42]: Maybe we can add a `FormFields.viewCheckboxNotChecked(fieldName = "enableAUIPreload");`

for https://issues.liferay.com/browse/LPS-112311?focusedCommentId=2160988&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2160988

**Comment by zhangyan0701** [comment]
Hi @john-co 

Thanks for reviewing the test!

Because below two reasons, I didn't update test:

1. Macro SystemSettings.configureSystemSetting asserts the checkbox is not checked after save configuration, see this line [SystemSettings.macro#L51](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/SystemSettings.macro#L51), so there is no need to add macro FormFields.viewCheckboxNotChecked.

2. This test needs to go to System Settings, but there is no system settings for virtual instance. So it has to skip running on virtual instance for this test.

And I reproduced [LPS-113623](https://issues.liferay.com/browse/LPS-113623) again on master head on Windows + Chrome/FF. And test also failed on CI Linux environment.
Please refer to the [Poshi report](https://testray.liferay.com/reports/production/logs/test-1-10/1590559442366/test-portal-acceptance-pullrequest(master)/3086/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.UIInfrastructureUsecase_CheckAUIPreloadConfiguration/index.html.gz).

**Comment by john-co** [comment]
pushed to https://github.com/john-co/liferay-portal/pull/304

**Pull request closed without merging**